src/bundle/Resources/public/scss/_main-menu.scss

```diff
@@ -33,14 +28,13 @@
 
         &--second-level {
             width: calculateRem(220px);
-            background-color: rgba($ibexa-color-dark, 0.8);
-            border-left: calculateRem(1px) solid $ibexa-color-info-800;
-            transition: all $ibexa-admin-transition-duration;
+            overflow-x: hidden;
+            overflow-y: auto;
```

Comment:
Little nitpick/useless knowledge - if one of the overflows is `hidden` and the second one is `visible` (which is default value) the second one will be automatically converted to `auto` ;)

ref: https://www.brunildo.org/test/Overflowxy2.html

Reply:
Hmm, so basically, no need to state `overflow-y: auto;` explicitly?